### ISSUE TITLE
launch arguments not being correctly passed when using "Run As Administrator"

### ISSUE THREAD

**Issue by IGP** [Flow-Launcher/Flow.Launcher#1182]
**Describe the bug/issue**
When using "Run As Administrator" launch arguments (see Figure 1) from the shortcut being launched do not appear to be maintained. These same arguments appear to be correctly passed through when NOT using "Run As Administrator". More specifically this is at least the 'Target' field and potentially the 'Start in' field on the shortcut.

**To Reproduce**
I used Discord to reproduce this via the default Discord shortcut available on the start menu. 

1. Have Discord installed and confirm the shortcut in your start menu is similar to the one in Figure 1 (version/path may be different)
2. In Flow navigate to Discord then do "Launch As Administrator"
3. Note that the UAC prompt is missing the arguments from the shortcut (see Figure 2)
4. Close the UAC dialog and attempt to launch the shortcut as Administrator from the Windows Start menu
5. Note that the UAC prompt correctly has the arguments from the shortcut (see Figure 3)

**Screenshots**
Figure 1: 
![image](https://user-images.githubusercontent.com/920452/166997127-cc497354-abce-4ba2-b2a4-22fdc2177e81.png)

Figure 2:
![image](https://user-images.githubusercontent.com/920452/167000096-9c079e0a-c007-403c-8ff9-8d95e348200a.png)

Figure 3:
![image](https://user-images.githubusercontent.com/920452/167000708-0a134e8a-6276-427d-87ea-eedd5a1d6602.png)

**Your System**
```
Windows build number: Microsoft Windows [Version 10.0.19044.1586]
Windows language: EN-US
Flow Launcher version: 1.9.3
```
**Flow Launcher Error Log**
No error appears to be actually logged but here is the log anyways:

[2022-05-05.txt](https://github.com/Flow-Launcher/Flow.Launcher/files/8633851/2022-05-05.txt)

**Comment by 1234Brian** [comment]
👍

**Comment by IGP** [comment]
bump this is definitely still occuring

**Comment by 1234Brian** [comment]
👍🌟

**Comment by nachmore** [comment]
I've root caused this to us not extracting arguments when a link resolves to an exe - essentially missing a call to `((IShellLinkW)link).GetArguments`).

I'll look into getting a PR for this one over the next couple of days.

**Comment by jjw24** [comment]
Awesome, thank you @nachmore

**Comment by IGP** [comment]
bumping so it doesn't get auto closed due to stale

**Comment by VictoriousRaptor** [comment]
Fixed in 1.10.0
